src/components/__tests__/HelpButton.cy.js

```diff
@@ -0,0 +1,217 @@
+        props: {
+          color: 'grey-10',
+          size: '13px',
+        },
+      });
+      cy.viewport('macbook-16');
+    });
+
+    it('renders help button', () => {
+      cy.window().then(() => {
+        cy.dataCy('button-help').should('be.visible');
+        cy.dataCy('button-help')
+          .should('have.css', 'font-size', '13px')
+          .should('have.css', 'font-weight', '500')
+          .should('have.backgroundColor', '#212121')
+          .should('have.css', 'border-radius', '50%'); // round
+        cy.dataCy('button-help').should('contain', 'question_mark');
+      });
+    });
+
+    it('renders help button with correct size', () => {
+      cy.dataCy('button-help').should('be.visible');
+      cy.dataCy('button-help').invoke('height').should('be.equal', 39);
+      cy.dataCy('button-help').invoke('width').should('be.equal', 39);
+    });
+
+    it('renders help icon', () => {
+      cy.dataCy('icon-help')
+        .should('contain', 'question_mark')
+        .should('have.color', '#fff');
+      cy.dataCy('icon-help')
+        .invoke('height')
+        .should('be.gt', 22)
+        .should('be.lt', 24);
+      cy.dataCy('icon-help')
+        .invoke('width')
+        .should('be.gt', 22)
+        .should('be.lt', 24);
+    });
+  });
+
+  context('button primary', () => {
+    beforeEach(() => {
+      cy.mount(HelpButton, {
+        props: {
+          color: 'primary',
+          size: '13px',
+        },
+      });
+      cy.viewport('macbook-16');
+    });
+
+    it('renders help button', () => {
+      cy.window().then(() => {
+        cy.dataCy('button-help').should('be.visible');
+        cy.dataCy('button-help')
+          .should('have.css', 'font-size', '13px')
+          .should('have.css', 'font-weight', '500')
+          .should('have.backgroundColor', `${colorPrimary}`)
+          .should('have.css', 'border-radius', '50%'); // round
+        cy.dataCy('button-help').should('contain', 'question_mark');
+      });
+    });
+
+    it('renders help button with correct size', () => {
+      cy.dataCy('button-help').should('be.visible');
+      cy.dataCy('button-help').invoke('height').should('be.equal', 39);
+      cy.dataCy('button-help').invoke('width').should('be.equal', 39);
+    });
+
+    it('renders help icon', () => {
+      cy.dataCy('icon-help')
+        .should('contain', 'question_mark')
+        .should('have.color', '#fff');
+      cy.dataCy('icon-help')
+        .invoke('height')
+        .should('be.gt', 22)
+        .should('be.lt', 24);
+      cy.dataCy('icon-help')
+        .invoke('width')
+        .should('be.gt', 22)
+        .should('be.lt', 24);
+    });
+  });
+
+  context('button small', () => {
+    beforeEach(() => {
+      cy.mount(HelpButton, {
+        props: {
+          color: 'grey-10',
+          size: '8px',
+        },
+      });
+      cy.viewport('macbook-16');
+    });
+
+    it('renders help button', () => {
+      cy.dataCy('button-help').should('be.visible');
```

Comment:
Same as my comment above https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/51/files#r1364873399.

Reply:
Edited.